Assets/Scripts/Flashlight.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Flashlight : MonoBehaviour
+{
+    private Light flashLight;
+    private bool lightOn = false;
+   
+        void Start()
+        {
+            flashLight = GetComponentInChildren<Light>();
+        }
+
+        void Update()
```

Comment:
I would recommend extracting the code inside update to it's own function, only keeping the Input.GetKeyDown if statement inside Update, and calling the new method from there.